docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[uptime-set-up]]
+= Set up monitors
+
+++++
+<titleabbrev>Set up monitors</titleabbrev>
+++++
+
+When setting up an uptime monitor, you will use a provided Docker image to run the monitor and send data back to Elasticsearch. 
+There are two approaches to configuring the monitor and connecting it to Elasticsearch.
+
+After set up is complete, the data will be available in the {uptime-app} in {kib} to view, analyze, and alert on.
+
+[discrete]
+[[uptime-set-up-choose]]
+== Choose an approach
+
+There are two ways to configure uptime monitors and connect them to the {stack}:
+
+* Use a *{heartbeat}* configuration file
+* beta[] Use *{agent}*, which runs Beats behind the scenes
+
+**{heartbeat}** is a lightweight daemon that you install on a remote server to periodically
+check the status of your services and determine whether they are available.
+Performance data is then gathered and reported back to Heartbeat,
+where it is formatted and sent to the Elastic Stack.
+
+// What are the pros and cons?
+
+beta[] *{agent}* is a single, unified agent that you can deploy to hosts or containers to collect data and send it to the {stack}.
```

Comment:
Should we be more explicit about recommending the agent? Then when we have the service we can more explicitly recommend the service.

Reply:
I added some details below, but yeah, I'd hang back a bit there for now, till we have https://github.com/elastic/uptime/issues/441